packages/cli/cloud/src/create-project/action.ts

```diff
@@ -0,0 +1,61 @@
+import inquirer from 'inquirer';
+import { AxiosError } from 'axios';
+import { defaults } from 'lodash/fp';
+import type { CLIContext, ProjectAnswers, ProjectInput } from '../types';
+import { tokenServiceFactory, cloudApiFactory, local } from '../services';
+
+function handleError(ctx: CLIContext, error: Error) {
+  const tokenService = tokenServiceFactory(ctx);
+  const { logger } = ctx;
+
+  logger.debug(JSON.stringify(error));
+  if (error instanceof AxiosError) {
+    switch (error.response?.status) {
+      case 401:
+        logger.error('Your session has expired. Please log in again.');
+        tokenService.eraseToken();
+        return;
+      case 403:
+        logger.error(
+          error.response.data ||
+            'You do not have permission to create a project. Please contact support for assistance.'
```

Comment:
I don't know what cases this can happen, but it there any additional information that can be provided on the cloud api side to reduce support requests in some cases? This feels like it could be a painful thing for support 😆

Reply:
Yes, users can use - - debug flag when running the command and send the logs to support. It should be sufficient to troubleshoot. At least for the first version 😅

Reply:
When you try to create a project with the free trial but you have already used it, API will throw a 403.
In the response.data we should have the appropriate message:
`'Unfortunately, you have already used your free trial for Strapi Cloud. Please contact our support team for more information.'`